packages/plugin-core/src/workspace.ts

```diff
@@ -383,6 +395,36 @@ export class DendronExtension {
     this.treeViews = {};
     this.webViews = {};
     this.setupViews(context);
+    this._typeRegistrar = new NoteTraitManager(new CommandRegistrar(context));
+
+    // Register any User Defined Note Types
+    const userTypesPath = vscode.workspace.workspaceFile
```

Comment:
with native workspaces, the root of workspace might not be the workspace file but where the dendorn.yml is. `getDWorkspace().wsRott` should take care of both cases

Reply:
also, `s/userType/userTraits`